src/main/java/com/redhat/cloud/notifications/RestValidationClient.java

```diff
@@ -25,4 +27,11 @@ public interface RestValidationClient {
     @CacheResult(cacheName = "baet-validation")
     Response validate(@RestQuery String bundle, @RestQuery String application, @RestQuery String eventType);
 
+    @GET
+    @Path("/certificate-according-bundle-and-app")
+    @Retry(maxRetries = 5)
+    @Produces(APPLICATION_JSON)
```

Comment:
If `@Produces` is here, we should also add `@Consumes`. They're not actually needed, but Quarkus did regress in the past in that area so it's not a bad thing to have them explictely declared.

Reply:
Because it's a get request, defining only `@Consumes` could be better.

Reply:
The annotations of a rest-client interface must reflect the behavior of the remote host, so if we only use one annotation it should be `@Produces` in this case.

But it really doesn't matter here because both `@Produces` and `@Consumes` are automatically added when the app depends on `quarkus-rest-client-jackson`.

Reply:
Interesting, let's add both annotations for a better understanding.